pkg/agent/openflow/network_policy.go

```diff
@@ -248,16 +256,28 @@ func (m *conjunctiveMatch) generateGlobalMapKey() string {
 			// To normalize the key, set full mask while a single port is provided.
 			valueStr = fmt.Sprintf("%d/65535", bitRange.Value)
 		}
+	case *int32:
+		// This cases include the matchValue is ICMPType or ICMPCode.
+		valueStr = fmt.Sprintf("%d", *m.matchValue.(*int32))
```

Comment:
Thanks for pointing it out. Yes, It will crash if `matchValue` is a nil *int32. It didn't crash now because `matchValue` is a nil interface{} instead of a nil *int32 to `matchValue` which will fall into the default case.